bool accelValid
bool fusionPoseValid
bool fusionQPoseValid
bool gyroValid
bool compassValid
bool readValid
